### DISCUSSION TITLE
Completed Solidity101! A huge thank you!

### DISCUSSION THREAD

**Discussion by PhilippeOurselin** [Cyfrin/foundry-full-course-cu#2356] [Show and tell]
Just completed the whole *Beginner courses*! This was an amazing journey, and I'm so excited to continue with the intermediate ones! So here is a little Solidity code to send my warmest thanks to all Cyfrin Staff, and especially @PatrickAlphaC !

```solidity
contract ThankYou {
    string internal message;

    constructor() {
        message = "A huge thank you to all the amazing people who are helping me learn Solidity, especially Patrick Collins, who has great pedagogical skills!";
    }

    function giveHugeThanks() public view returns(string memory) {
        return message;
    }
}
```
![solidity-completed](https://github.com/user-attachments/assets/0260609a-0d5d-48b0-9861-6d9c12124541)

**Comment by EngrPips** [comment]
Hey @PhilippeOurselin, Keep crushing and that is a nice way to inform us that you truly understood everything that was taught in the course.
